pombola/south_africa/static/js/mp-profiles-live-filter.js

```diff
@@ -39,7 +39,19 @@ var showFilterResults = function showFilterResults(){
     return $results;
 };
 
+var analyticsEventSent = false;
+
 var filterProfiles = function filterProfiles(searchTerm, filterer){
+
+    if (!analyticsEventSent) {
+        ga('send', 'event', {
```

Comment:
Why not use `window.analytics.trackEvent` from `js/analytics.js` (which gets concatenated into `base.js` by `pombola/base.py`)?

```javascript
window.analytics.trackEvent({
  eventCategory: 'mp-name-search',
  eventAction: 'search'
});
```

It’s what we used in the homepage A/B test, so I think